Demo/WalletLibraryDemo/WalletLibraryDemo/VerifiedIdRepository.swift

```diff
@@ -8,20 +8,54 @@ import WalletLibrary
 
 struct VerifiedIdRepository {
     
-    /// TODO: add persistent storage.
-    var storedVerifiedIds: [VerifiedId] = []
+    let verifiedIdClient: VerifiedIdClient
     
-    mutating func save(verifiedId: VerifiedId) throws {
-        storedVerifiedIds.append(verifiedId)
+    let store = PersistenceController.shared.container
+    
+    init(verifiedIdClient: VerifiedIdClient) {
+        self.verifiedIdClient = verifiedIdClient
+    }
+    
+    func save(verifiedId: VerifiedId) throws {
+        let rawVerifiedId = try verifiedIdClient.encode(verifiedId: verifiedId).get()
+        let viewContext = store.viewContext
+        let newVerifiedId = RawVerifiedId(context: viewContext)
+        newVerifiedId.raw = rawVerifiedId
+        try viewContext.save()
     }
     
-    mutating func delete(verifiedId: VerifiedId) throws {
-        storedVerifiedIds.removeAll {
-            $0.id == verifiedId.id
+    func delete(verifiedId: VerifiedId) throws {
+        let viewContext = store.viewContext
+        let request = NSFetchRequest<NSFetchRequestResult>(entityName: "RawVerifiedId")
```

Comment:
nit: may be name it as rawVerifiedId? It is confusing with request object.

Reply:
it is actually a database request.